Makefile

```diff
@@ -320,7 +324,7 @@ test-yaml:
 .PHONY: test-shfmt
 test-shfmt:
 	@which shfmt >/dev/null 2>&1 || (echo "Command 'shfmt' not found, can not execute bash script syntax checks" && false)
-	shfmt -d -i 4 -bn -ci -sr $$(shfmt -f .)
+	shfmt -d -i 4 -bn -ci -sr $$(shfmt -f . | grep --invert-match node_modules)
```

Comment:
How about the same approach as for "test-shellcheck" to explicitly select shell files by type

Reply:
I didn't like the `shfmt -f .`-approach anyway, e.g. in my unclean repo it just stopped after a permission error and thus didn't check much. I made it a separate commit.